daemon/rpc_account.go

```diff
@@ -30,14 +57,21 @@ func (r *RPC) AccountInfo(ctx context.Context, _ *pb.Empty) (*pb.AccountResponse
 		accountInfo.Type = internal.CodeSuccess
 	}
 
-	dedicatedIPExpired, err := r.ac.IsDedicatedIPExpired()
+	accountInfo.DedicatedIpStatus = internal.CodeSuccess
+	dipServices, err := r.ac.GetDedicatedIPServices()
 	if err != nil {
-		log.Println(internal.ErrorPrefix, "checking dedicated IP expiration: ", err)
+		log.Println(internal.ErrorPrefix, "getting dedicated ip services: %w", err)
 		return &pb.AccountResponse{Type: internal.CodeTokenRenewError}, nil
```

Comment:
If the user doesn't have DIP subscription it will always return and error.

Reply:
If user doesn't have DIP subscription `GetDedicatedIPServices` should return an empty list. Only if user has DIP subscriptions but no DIP servers for any of them it should return an error. Unless something is implemented wrong. :thinking:

Reply:
sorry the error is from `findLatestDIPExpirationData` a few lines bellow. The function returns error when no servers are available. And since DIP subscription is missing user has no servers